libs/category/src/index.ts

```diff
@@ -0,0 +1,9 @@
+export { DaffCategory } from './models/category';
+export * from './actions/category.actions';
+
+import * as fromCategory from './reducers/index';
```

Comment:
[Barrels don't that well in exports](https://github.com/ng-packagr/ng-packagr/issues/382#issuecomment-362318700). We should rethink this.

Reply:
I made a lot of changes to how reducers and selectors are organized, and I think it makes a lot more sense. Also got rid of barrel imports